src/sys/inotify.rs

```diff
@@ -235,14 +235,8 @@ impl Inotify {
     }
 }
 
-impl AsRawFd for Inotify {
-    fn as_raw_fd(&self) -> RawFd {
-        self.fd
-    }
-}
-
```

Comment:
This trait impl extracts the `RawFd` from `struct Inofity`, IMO, this is used to `close(2)` the file descriptor representing the `inotify` instance, which is no longer needed as `OwnedFd` will drop it automatically.